pkg/images/images_test.go

```diff
@@ -99,8 +99,8 @@ func Test_toKeep(t *testing.T) {
 		{
 			name: "min < 2.7.0 < max < 2.9.0",
```

Comment:
we need to update the name here based on the updated rancher versions
```suggestion
			name: "min < 2.8.0 < max < 2.10.0",
```